modules/playlists/src/main/java/org/opencastproject/playlists/PlaylistRestService.java

```diff
@@ -0,0 +1,771 @@
+      sortCriterion = SortCriterion.parse(optSort.get());
+
+      switch (sortCriterion.getFieldName()) {
+        case "updated":
+          break;
+        default:
+          logger.info("Unknown sort criteria {}", sortCriterion.getFieldName());
+          return Response.status(SC_BAD_REQUEST).build();
+      }
+    }
+
+    List<JaxbPlaylist> jaxbPlaylists = new ArrayList<>();
+    for (Playlist playlist : service.getPlaylists(limit, offset, sortCriterion)) {
+      jaxbPlaylists.add(new JaxbPlaylist(playlist));
+    }
+
+    return Response.ok().entity(new GenericEntity<>(jaxbPlaylists) { }).build();
+  }
+
+  @GET
+  @Produces(MediaType.APPLICATION_XML)
+  @Path("playlists.xml")
+  @RestQuery(
+      name = "playlists",
+      description = "Get playlists. Playlists that you do not have read access to will not show up.",
+      returnDescription = "A XML object containing an array.",
+      restParameters = {
+          @RestParameter(
+              name = "limit",
+              isRequired = false,
+              type = INTEGER,
+              description = "The maximum number of results to return for a single request.",
+              defaultValue = "100"
+          ),
+          @RestParameter(
+              name = "offset",
+              isRequired = false,
+              type = INTEGER,
+              description = "The index of the first result to return."
+          ),
+          @RestParameter(
+              name = "sort",
+              isRequired = false,
+              type = STRING,
+              description = "Sort the results based upon a sorting criteria. A criteria is specified as a pair such as:"
+                  + "<Sort Name>:ASC or <Sort Name>:DESC. Adding the suffix ASC or DESC sets the order as ascending or"
+                  + "descending order and is mandatory. Sort Name is case sensitive. Supported Sort Names are 'updated'"
+              ,
+              defaultValue = "updated:ASC"
+          ),
+      },
+      responses = {
+          @RestResponse(responseCode = SC_OK, description = "A playlist as XML."),
+      })
+  public Response getPlaylistsAsXml(
+      @FormParam("limit") int limit,
+      @FormParam("offset") int offset,
+      @FormParam("sort") String sort)
+          throws NotFoundException {
+    return getPlaylistsAsJson(limit, offset, sort);
+  }
+
+  @POST
+  @Produces(MediaType.APPLICATION_JSON)
+  @Path("new.json")
+  @RestQuery(
+      name = "create",
+      description = "Creates a playlist.",
+      returnDescription = "The created playlist.",
+      restParameters = {
+          @RestParameter(
+              name = "playlist",
+              isRequired = false,
+              description = "Playlist in JSON format",
+              type = TEXT,
+              jaxbClass = JaxbPlaylist.class,
+              defaultValue = SAMPLE_PLAYLIST_JSON
+          )
+      },
+      responses = {
+          @RestResponse(responseCode = SC_OK, description = "Playlist updated."),
```

Comment:
I think this method should return the status code [201 Created](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/201).